src/main/java/com/exasol/releasedroid/adapter/github/GitHubRepositoryValidator.java

```diff
@@ -7,21 +7,22 @@
 import java.util.logging.Logger;
 
 import com.exasol.errorreporting.ExaError;
-import com.exasol.releasedroid.repository.ReleaseLetter;
-import com.exasol.releasedroid.usecases.Repository;
+import com.exasol.releasedroid.adapter.AbstractRepositoryValidator;
 import com.exasol.releasedroid.usecases.report.Report;
 import com.exasol.releasedroid.usecases.report.ValidationResult;
+import com.exasol.releasedroid.usecases.repository.ReleaseLetter;
+import com.exasol.releasedroid.usecases.repository.Repository;
 
 /**
  * Contains validations for a Git project.
  */
-public class GitRepositoryValidator extends AbstractRepositoryValidator {
-    private static final Logger LOGGER = Logger.getLogger(GitRepositoryValidator.class.getName());
+public class GitHubRepositoryValidator extends AbstractRepositoryValidator {
```

Comment:
It's not a GitHub repository validator. It also validates a local Git repository